styles/text.scss

```diff
@@ -109,8 +109,23 @@ button:focus-visible {
 
 /* Inline code blocks */
 p > code,
-li > code {
-  @apply border border-gray-40 dark:border-gray-80 text-red-70 rounded-md px-1 mx-1 break-words;
+li > code,
+/* Inline code blocks in docstrings */
+tt.docutils.literal {
+  @apply break-words px-1 rounded;
+  @apply text-green-80 bg-gray-10;
+  @apply dark:text-green-50 dark:bg-gray-90;
+  @apply text-base;
+}
+
+/* Improve `code` links. Don't use border, use underline instead. */
+p > a:has(tt.docutils.literal) {
+  @apply border-0;
+}
+
+/* Improve `code` links. Don't use border, use underline instead. */
+p > a > tt.docutils.literal {
+  @apply underline;
```

Comment:
If we use underline, commands with underscores will look like this:
<img width="291" alt="image" src="https://github.com/user-attachments/assets/5b0b9701-6fb4-4f37-9dc4-942da12fb5f3" />

Also, the site uses border for links elsewhere, so if we do switch it, we have other styling to correct too. This is what the preview currently shows:

A link in the function description of st.logo:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/8caf14d8-74cd-4ba6-85e1-96b1143b34da" />

A link in the class description of SQLConnection:
<img width="788" alt="image" src="https://github.com/user-attachments/assets/76afbfdf-f723-47e8-80f0-37293ce6ccdd" />

Links from the connections concept page (both standard and header-code):
<img width="793" alt="image" src="https://github.com/user-attachments/assets/c8c252bc-b18d-4815-8a38-1d305bc3c33e" />


The only thing that's been changed is inline code within a parameter description, I think.

Reply:
I'd vote for leaving it as a border to be consistent across all links and just modify the color if that's what you want. Though I'd be a little confused by the color scheme if we make it a green underline for code (to always match the text color), then how do we handle the hover effect. We currently hover black -> red for the bottom border on links, but green -> red feels....seasonal. 😅

Reply:
For regular links we'll continue to use the existing design. What we're changing here is the design of links inside inline code from autofunctions.

So from your examples, only the SQL Connection example will be impacted by this PR. The rest will stay the same.

Reply:
In the connection to data article, `st.connection` is also inline code; it's just not inline code within Autofunction. Do you mean to exclude that as well?